.changeset/rare-shiny-bulbasaur.md

```diff
@@ -0,0 +1,29 @@
+---
+"@comet/admin": minor
+---
+
+Add the ContentTranslationServiceProvider which can be used to wrap components that use FinalFormInput (and later also RTE) to enable the activation of the translation feature.
+
+```tsx
+<ContentTranslationServiceProvider
+    enabled={true}
+    translate={async function (input: string): Promise<string> {
+        return yourTranslationFnc(value);
+    }}
```

Comment:
This could wouldn't work, because you're using input and value. Change both to text please